src/multicall/MulticallExecutor.ts

```diff
@@ -23,9 +23,12 @@ export class MulticallExecutor {
     }
     const provider = await this.providerSource()
 
-    const result = await provider.call(tx)
-
-    return result
+    try {
+      const result = await provider.call(tx)
+      return result
+    } catch (e) {
```

Comment:
Suppressing the error doesn't seem wise to me.  Can we gather the errors up instead and throw them in one go?

Reply:
Could we use `console.warn()` to let developers know that that's what happened, but not throw an exception? I'd argue that it's not an exception if a call gets interrupted mid-flight.